Algorithmia/handler.py

```diff
@@ -0,0 +1,98 @@
+import sys
+import json
+import base64
+import traceback
+import six
+
+
+class Handler(object):
+
+    def __init__(self, apply_func, load_func=None):
+        """
+        Creates the handler object
+        :param apply_func: A required function that can have an arity of 1-2, depending on if loading occurs
+        :param load_func: An optional supplier function used if load time events are required, has an arity of 0.
+        """
+        self.FIFO_PATH = "/tmp/algoout"
+        self.apply_func = apply_func
+        if load_func:
+            self.load_func = load_func
+        else:
+            self.load_func = lambda: None
```

Comment:
Minor nit: could we just do `def __init__(self, apply_func, load_func=lambda: None):`